app.js

```diff
@@ -38,12 +38,12 @@ app.use(require('./middlewares'));
 app.use(require('./controllers'));
 
 // Basic 404 handler
-app.use(function(req, res) {
+app.use((req, res) => {
```

Comment:
Don't know if it's possible to break this out now, but this PR seems to have all changes from #37 in it as well.